build/stf-run-ci/defaults/main.yml

```diff
@@ -37,6 +37,8 @@ __smart_gateway_bundle_image_path:
 default_operator_registry_image_base: registry.redhat.io/openshift4/ose-operator-registry
 default_operator_registry_image_tag: v4.12
 
+elastic_version: 7.16.1
```

Comment:
Nitpicky, but probably should be elasticsearch_version since Elastic is technically the company name for the product Elasticsearch. I would look the other way though.

Reply:
```suggestion
elasticsearch_version: 7.16.1
```